README.md

```diff
@@ -431,6 +433,41 @@ Time.assertEqual(
 Time.run(err => console.error(err));
 ```
 
+You can optionally pass a custom comparator function. This is useful if you want to do things like testing your DOM with tools such as [html-looks-like](https://github.com/staltz/html-looks-like).
+
```

Comment:
Can we document that the comparator function can either return a bool or throw an error if something is wrong? This is useful as it allows the comparator to provide a custom error message.